apps/web/pages/apps/installed/[category].tsx

```diff
@@ -112,37 +106,95 @@ interface IntegrationsListProps {
   handleDisconnect: (credentialId: number) => void;
 }
 
-const IntegrationsList = ({ data, handleDisconnect }: IntegrationsListProps) => {
+const IntegrationsList = ({ data, handleDisconnect, variant }: IntegrationsListProps) => {
+  const { data: usersMetadata } = trpc.viewer.getUserMetadata.useQuery();
+  const utils = trpc.useContext();
+
+  const [locationType, setLocationType] = useState<(EventLocationType & { slug: string }) | undefined>(
+    undefined
+  );
+
+  const updateDefaultAppMutation = trpc.viewer.updateUserDefaultConferencingApp.useMutation({
+    onSuccess: () => {
+      showToast("Default app updated successfully", "success");
+      utils.viewer.getUserMetadata.invalidate();
+    },
+  });
+
+  const { t } = useLocale();
   return (
-    <List>
-      {data.items
-        .filter((item) => item.invalidCredentialIds)
-        .map((item) => (
-          <AppListCard
-            key={item.name}
-            description={item.description}
-            title={item.name}
-            logo={item.logo}
-            isDefault={item.isGlobal}
-            shouldHighlight
-            slug={item.slug}
-            invalidCredential={item.invalidCredentialIds.length > 0}
-            actions={
-              <div className="flex  justify-end">
-                <ConnectOrDisconnectIntegrationButton
-                  credentialIds={item.credentialIds}
-                  type={item.type}
-                  isGlobal={item.isGlobal}
-                  installed
-                  invalidCredentialIds={item.invalidCredentialIds}
-                  handleDisconnect={handleDisconnect}
-                />
-              </div>
-            }>
-            <AppSettings slug={item.slug} />
-          </AppListCard>
-        ))}
-    </List>
+    <>
+      <List>
+        {data.items
+          .filter((item) => item.invalidCredentialIds)
+          .map((item) => {
+            const appSlug = item?.slug;
+            const appIsDefault =
```

Comment:
If the app equals the users default 

if no default is set we default to daily-video